File: cloud/dynamic-site/gt-cs-1331-py/app/gtcs1331/solutions/solution108.py
'''Solution 1.8 implementation'''
import math
from gtcs1331.solution_api.solution import Solution

class Solution108(Solution):
    '''Solution 1.8'''
    id = "1.8"
    title = "Circle measure"

    def __answer(self):
        rds = 5.5
        prmr = 2 * rds * math.pi
        area = rds * rds * math.pi

        return [
            f"perimeter = {prmr:.6f}",
            f"area = {area:.6f}"]

    answer = property(__answer)

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution108.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
    public class Solution108 : ImmutableSolution {
        public Solution108() :
            base("1.8", "Circle measure") {
        }

        public override IEnumerable<string> Answers {
            get {
                double rds = 5.5;
                double prmr = 2 * rds * Math.PI;
                double area = rds * rds * Math.PI;

                return new string[] {
                    $"perimeter = {prmr:f6}",
                    $"area = {area:f6}"
                };
            }
        }
    }
}